hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -55,6 +55,8 @@
   private final String rolloverLogWriteToken;
   private FSDataOutputStream output;
   private boolean closed = false;
+  private transient Thread shutdownThread = null;
```

Comment:
is there a reason why this should be `transient`. we dont serialize this, right

Reply:
Yes, This is just to be defensive .